config/opal_summary.m4

```diff
@@ -92,6 +93,7 @@ AC_DEFUN([OPAL_SUMMARY_PRINT],[
 Open MPI configuration:
 -----------------------
 Version: $OMPI_MAJOR_VERSION.$OMPI_MINOR_VERSION.$OMPI_RELEASE_VERSION$OMPI_GREEK_VERSION
+MPI Standard Version: $MPI_VERSION.$MPI_SUBVERSION
```

Comment:
If we're printing this here, why do we need the print in OPAL_SAVE_MPI_STANDARD_VERSION?  That seems redundant...

Reply:
It is redundant, but we also print the OMPI version and release information in `OPAL_SAVE_VERSION` around the same time. I think it's ok for the summary to be redundant for a few items like this.

Reply:
yeah, I guess...  It's just that we can't both complain about the runtime of configure and keep doing things that make it take longer to run.  I'm fine here, but we're going to have to address this at some point.